cmd/compgen/cmd/receiver/receiver_test.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRun(t *testing.T) {
+	// Missing args. Note: this is normally enforced by cobra
```

Comment:
Why does "this is normally enforced by cobra" mean to the reader of the code. Does that impact the tests?

Reply:
ok, yea i'd just explain the comment, tell the why behind it.